mappings/net/minecraft/data/server/recipe/CookingRecipeJsonFactory.mapping

```diff
@@ -8,29 +8,51 @@ CLASS net/minecraft/unmapped/C_ffzmwtoh net/minecraft/data/server/recipe/Cooking
 	FIELD f_vfhkylcl cookingTime I
```

Comment:
wait heck, I might have stepped on your toes here with PR #354

Reply:
did we both do the param busywork :waaaa:

Reply:
wait nvm, i only stepped on your toes partially; i didn't directly map those (i was recalling the recipe remap work) but it does mean it'll need to be changed to the new `ingredient`/`result` terminology

Reply:
we can do that! I'll get on it